packages/gestalt/src/SideNavigation.css

```diff
@@ -14,13 +15,39 @@
 
 .fullHeight {
   height: 100%;
-  overflow: scroll;
+  overflow-y: scroll;
+}
+
+.contentWidthTransition {
+  transition: width 0.2s;
 }
 
 .section {
+  margin-bottom: var(--space-600);
```

Comment:
`Section` now has both top and bottom margin. It was done mainly for the purpose of visially splitting `Section` from `TopItem` and `Group` (these don't have any kind of margin). 

It is for the case where `Section` comes before `TopItem` or `Group`. If `Section`s are stacked, the top and bottom margins collapse, and the gap between them stays consistent.